framework/math/ParallelVector/parallel_vector.h

```diff
@@ -0,0 +1,157 @@
+#ifndef PARALLEL_VECTOR_H
+#define PARALLEL_VECTOR_H
+
+#include "chi_runtime.h"
+
+#include <vector>
+#include <cstdint>
+#include <map>
+
+#include <mpi.h>
+
+
+namespace chi_math
+{
+
+enum class OperationType : short
```

Comment:
Forgot about this one previous round. Can you please make this `VecOpType` or something that ties it to vectors. Right now this will land in `chi_math::OperationType` which could easily clash with something in future or be very ambiguous.